ci/pod/docker-compose.plugin.yml

```diff
@@ -225,6 +225,66 @@ services:
       http.port: 9201
       xpack.security.enabled: 'true'
 
+  elasticsearch-noauth-2:
```

Comment:
Running so many Elasticsearch containers simultaneously in CI may strain computing resources, and causing tests to fail.

Reply:
i removed two of these containers(not needed)